pkg/osx/req_pyobjc.txt

```diff
@@ -29,29 +29,28 @@ pyobjc-framework-coreaudio==6.2.2  # via pyobjc, pyobjc-framework-coreaudiokit
 pyobjc-framework-coreaudiokit==6.2.2  # via pyobjc
 pyobjc-framework-corebluetooth==6.2.2  # via pyobjc
 pyobjc-framework-coredata==6.2.2  # via pyobjc, pyobjc-framework-cloudkit, pyobjc-framework-syncservices
-pyobjc-framework-corehaptics==6.2.2  # via pyobjc
```

Comment:
@twangboy looks like `pyobjc-framework-corehaptics` is missing on 10.15 and causes the minion to crash.

Reply:
after a quick search, this appears to be the only one that is missing now.

Reply:
I guess I missed one... Thanks!